.github/workflows/update-lts-tags.yaml

```diff
@@ -0,0 +1,54 @@
+name: Update LTS Tags
+
+on:
+  schedule:
+    - cron: '0 0 * * 1-5' # At 00:00 on every day-of-week from Monday through Friday
```

Comment:
Do we have enough change for daily release? I prefer weekly release e.g. Raised PR Sunday night > Dos review on Monday morning

Reply:
I think `0 0 * * SUN` does make sense. Once a week on Sunday, so that if there are any new LTS tags, we have a PR to review on Monday.